### DISCUSSION TITLE
Piazzetta P230M nothing happened

### DISCUSSION THREAD

**Discussion by comperta** [philibertc/micronova_controller#27] [Q&A]
Hello @philibertc ,
I have a Piazzetta P230 M with a wifi module. from what I see the wifi module is connected to the SERIAL (GND + Serial + 20v => the 5v is not used) and that has a panel on the edge of the stove. 
On this panel I find a serial connector (+ 5V, + 16v, serial ?, GND).
I tried to make the circuit myself and connect it to this connector, but obviously there is no response from the stove. 
I use test2bis too and nothing happened.
how can I check this ?
![16403464960332857817538357080428](https://user-images.githubusercontent.com/21957307/147350293-bcefc2f2-d025-458d-9910-50d06bb192bd.jpg)

**Comment by philibertc** [comment]
> GND + Serial + 20v => the 5v is not used

Hello @comperta 
The only pin you should not use in your circuit is the approx 20V.
Can you please annotate this schematic to show me how you connected the circuit:
![image](https://user-images.githubusercontent.com/57588282/147407519-b2d010f0-74c4-45ef-a32c-23c2ee109523.png)

Sincerely,
philibertc

**Reply by comperta** [reply]
Hello @philibertc ,

As requested the schematic annotate :-)
![147407519-b2d010f0-74c4-45ef-a32c-23c2ee109523__01](https://user-images.githubusercontent.com/21957307/147466539-b87b2301-e359-47cd-88be-1a689241bfd9.png)
![IMG_20211227_121246](https://user-images.githubusercontent.com/21957307/147466546-1aae22ab-81c0-4785-8677-e9da4e31bb78.jpg)
![IMG_20211227_121542](https://user-images.githubusercontent.com/21957307/147466667-da43be44-ed2e-433f-9837-a174a4f57269.jpg)

I also tried on a breadboard with another esp8266 and I get the same result, I probably made a connection error in the diagram but I do not see where :-(

as you can see the esp is configured well but it doesn't seem to read anything, how can i test the pc817?
![Screenshot_20211227-132305](https://user-images.githubusercontent.com/21957307/147471638-308cbe69-97b9-42ef-9aa6-052e777a02b9.jpg)

**Reply by philibertc** [reply]
Hello
You can test the optocouplers like this:
![image](https://user-images.githubusercontent.com/57588282/147656260-00146b0a-220d-4d15-bdd2-4eb64294e0c1.png)
With the multimeter in ohmmeter mode. You must measure a resistance of about 100 Ohms when you apply the 5V (via the resistor of course).
